apps/perf-test-react-components/src/scenarios/CheckboxShim.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
```

Comment:
I don't think we want to add shims to our performance tests.  Because they can use both v8 and v9, we don't set a performance bar for them.

Reply:
Thanks for your suggestion. I have removed the perf codes in this PR. 
For the perf of shims, most of the current shims in our team only use corresponding v9 components. IIRC, the v9 render time is half less than v8. Maybe we can set some perf bar for it because:
1. We are also do internal perf optimization of our shims.
2. We are still fix some bugs in shims, if some perf test goal is set, it may help US detect perf regression.